production/queries/create_courses_mv.sql

```diff
@@ -5,21 +5,36 @@ SELECT
     c.subject_code AS subject_code,
```

Comment:
Course details MV now stores all course detail fields (description, prereq course codes, etc), it's now used by both course listing and course detail SQL queries.